### DISCUSSION TITLE
Trilium vs Marginote/Liquidtext

### DISCUSSION THREAD

**Discussion by BradKML** [TriliumNext/Trilium#2028] [Ideas]
As seen in https://www.youtube.com/watch?v=uRlI1ZCNP5o there are closed source apps for Paperless research.

Trillium can do a mind map, but they can't do PDF highlighting/citations.
Adding these two into Trilium suite would be good:
- PDF annotator and note taker https://xournalpp.github.io/
- Citation tool https://www.zotero.org/

**Comment by Waxwood** [comment]
Trilium will display PDFs in the document window. This is very convenient, but when I want to edit or annotate the PDFs stored in Trilium, I just click on OPEN in external program and then use Qoppa's PDF Studio to annotate and edit PDFs. When I click on SAVE in PDF Studio, it saves the PDF which I can then display properly in Trilium. 

I particularly like Trilium, but am not sure if it needs a PDF editor. 
But Trilium most definitely does need integration with Zotero in order to become a serious tool for researchers. I checked out Zettlr and considered switching to it because it offers full integration with the open-source software Zotero. However, I found Zettlr to be quite weak compared to Trilium- I'm not sure why Zettlr is so popular. 

If Trilium had Zotero integration, and ability to add keywords to selections of text and thus acquire basic QDA (qualitative data research functions), it would become a formidable research tool indeed.

**Reply by BradKML** [reply]
Perhaps instead of having it as a feature, maybe a multi-download pack that has integration built-in?
And yes Zotero is really sweet.

**Reply by zadam** [reply]
@Waxwood could you describe what do you mean by "zotero integration"? I'm not familiar with this software.

**Reply by BradKML** [reply]
@zadam Zotero is an academic paper organizing and citation tool (and people have integrated PDF downloaders into this tool) https://zotero.org

**Reply by zadam** [reply]
Yes, I can see that. But to a person who does not have experience with such tool it's not really clear what workflows does it provide and how Trilium is supposed to integrate with it.

**Reply by BradKML** [reply]
The workflow would be:
1. Find articles/papers you do like
2. Add the article to Zotero, PDF downloader will save the PDF locally
3. Cite articles in Trilium Notes (e.g. Cheatsheets, Summary) for indexing
4. Extract a list of relevant articles/papers through Trilium
5. Zettlr and other Paper Writing Software will include citational features as well.
(https://docs.zettlr.com/en/academic/citations/)

**Reply by ghost** [reply]
@zadam Maybe he means something like this https://youtu.be/_Fjhad-Z61o  (demonstration from minute 16:48 to 18:00). If you want to know more about zotero see the whole video, it is pretty interesting.

I also think that an integration with a reference manager is a must have for any student-researcher. I have been waiting (and it looks others also) for an integration like that in trilium. 

Thanks for this amazing software!

**Reply by zadam** [reply]
@bambirombi thanks, that makes it clearer. Trilium can import markdown files with the annotations, but opening the links back to Zotero is currently not possible (blocked by https://github.com/ckeditor/ckeditor5/issues/8248)

**Reply by BradKML** [reply]
This is something similar for LogSeq https://github.com/aljedaxi/logseq-zotero/

**Comment by WantToLearnJapanese** [comment]
Opening Zotero URI seems possible now?
> @bambirombi thanks, that makes it clearer. Trilium can import markdown files with the annotations, but opening the links back to Zotero is currently not possible (blocked by [ckeditor/ckeditor5#8248](https://github.com/ckeditor/ckeditor5/issues/8248))
